protocol-units/settlement/mcr/contracts/script/DeployMCR.s.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity ^0.8.13;
+
+import "forge-std/Script.sol";
+import "../src/MCR.sol";
+
+contract DeployMCR is Script {
+    function run() external {
+        vm.startBroadcast();
+
+        MCR mcr = new MCR(
+            5, 
+            128,
+            100 ether, // should accumulate 100 ether
+            100 ether, // each genesis validator can stake up to 100 ether
+            0
+        );
+        vm.stopBroadcast();
+
+        // Comment because the Genesis ceremony works (Assert ok)
```

Comment:
Do we want to keep this in?

Reply:
I don't understand. From Slack discussion we decide to use forge script to manage smart contract init and not in Rust. It's your question?